app/components/UI/Ramp/Views/BuildQuote/BuildQuote.tsx

```diff
@@ -563,6 +564,12 @@ const BuildQuote = () => {
         location: screenLocation,
       };
 
+      trace({
+        name: TraceName.RampQuoteLoading,
+        tags: {
+          rampType: isBuy ? RampType.BUY : RampType.SELL,
```

Comment:
we have `rampType` in the SDK variables already
```suggestion
          rampType,
```